Adiciona guia de atividades (#30)
pugpe/meetup-cookbook: languages/portuguese/arquivos/ATIVIDADES.md

# Atividades
> Aqui conterá uma descrição sobre submissão de atividades, avaliação de atividades e confirmação de atividades. Desde como fazer até onde fazer o gerenciamento dessas etapas.
## Submissão de atividades

*Caso o evento for oferecer atividades que pessoas possam submeter (por exemplo: palestras, tutoriais, etc.)*

### Descrição

> A descrição da etapa de submissão pode fazer muita diferença, pois é o que pode atrair ou afastar pessoas interessadas. Por isso, faça uma descrição objetiva e clara, para que as pessoas não fiquem perdidas no processo.

Reviewer: ```suggestion
A descrição da etapa de submissão pode fazer muita diferença, pois é o que pode atrair ou afastar pessoas interessadas. Por isso, faça uma descrição objetiva e clara, para que as pessoas não fiquem perdidas no processo.

Caso você não tenha certeza sobre a claridade do texto, não hesite em compartilhar com outras pessoas para receber feedbacks. 
```

Reviewer 2: Irei modificar, obrigado! :)

Reviewer: Aguardando kkkk

Reviewer 2: Antes tarde do que nunca! Modificado

- Data e local do evento
- Breve descrição sobre como será o evento
- O que eles estão submetendo (ex.: tutorial, palestra, etc.)
    - Duração das atividades
    - Principais áreas de conhecimento que são aceitas
- Instruções de como deve ser feita a submissão
- Breve explicação do processo de avaliação e seleção
- Declaração sobre o Código de Conduta
- Divulgação dos canais de comunicação do grupo

Aqui temos um [modelo de descrição de submissão de palestras](languages/portuguese/arquivos/modelos/DESCRICAO-SUBMISSAO-PALESTRAS.md) que pode ser utilizado. E também um [modelo de descrição de votação de palestras](languages/portuguese/arquivos/modelos/DESCRICAO-VOTACAO-PALESTRAS.md), caso seja necessário.

### Plataforma

Algumas plataformas oferecem funcionalidades que facilitam bastante a vida de quem está organizando e de quem está submetendo.

Um breve cardápio de plataformas conhecidas:

#### [Google Forms](https://docs.google.com/forms)
- Submissão separada da votação (caso tenha votação)
- Sem votação pública:
    - Submissão anônima
- Com votação pública:
    - Submissão e votação separadas\*
    - Submissão anônima
    - Votação com escolha única, múltipla escolha, etc.

>     - Voto em escala, seleção, etc. 

Reviewer: Podia explicar melhor esses topicos? estava lendo agora e não consegui entender direito.

Reviewer 2: Mudei completamente, vê lá se ficou melhor

#### [Speakerfight](https://speakerfight.com/events/create/)

> - Open source e brasileira

Reviewer 3: ```suggestion
- Open source e Pernambucana
```

Reviewer 2: Acho que brasileira fica menos exclusivo. Fiquei pensando nisso na hora de colocar também

- Sem votação pública:
    - Submissão identificada ou anônima
    - Prazo de submissão
- Com votação pública:
    - Submissão e votação ao mesmo tempo
    - Submissão (e votação) identificada ou anônima
    - Votação com múltipla escolha
    - Voto na escala de 1 a 5 (carinha com raiva até carinha rindo)

>     - Prazo de submissão e votação

Reviewer: Mesma coisa aqui.

Acho que seria melhor colocar antes desses tópicos um:
`Possibilidades utilizando essa ferramenta` ou algo assim.

Reviewer 2: Mudei completamente também uwheuwheuahuewa

\* Um organizador deve preencher um novo formulário com o conteúdo das submissões e abrí-lo para que seja feita a votação com as atividades submetidas. É possível anonimizar a votação omitindo o nome do autor da atividade neste passo.

## Avaliação de atividades

Caso não seja feita votação pública, é interessante considerar algumas coisas na hora de avaliar e selecionar as atividades:
- Normalmente, é desejável um evento com atividades de diferentes níveis, para que tanto participantes experientes quanto novatos possam ter uma experiência legal
- Também é interessante não ter temáticas muito parecidas, para que o evento seja diversificado
- Dar voz a pessoas que nunca apresentaram num PUG também é legal
- O assunto da atividade é relacionado a Python? A comunidade Python é aberta a outras tecnologias, mas pode ser dada preferência a conteúdo relacionado à linguagem se quiser

> Se for feita votação pública, os interessados no evento que irão decidir o que querem ver :relaxed:. Mas se quiser, também podem ser utilizados alguns dos critérios anteriores para selecionar. Desde que isso esteja bem claro na descrição da submissão. É bom manter o nível de interferência ao mínimo quando houver votação pública.

Reviewer: ```suggestion
Se for feita votação pública, os interessados no evento que irão decidir o que querem ver :relaxed:. Mas se quiser, também podem ser utilizados alguns dos critérios anteriores para selecionar. Desde que isso esteja bem claro na [descrição de submissão](languages/portuguese/arquivos/modelos/DESCRICAO-SUBMISSAO-PALESTRAS.md). É bom manter o nível de interferência ao mínimo quando houver votação pública.
```

Reviewer 2: Modificado

Ah! E caso não tenha atividades suficientes para completar a programação, ignora tudo isso e coloca todas elas no evento :laughing:

## Confirmação de atividades

> Ao finalizar a avaliação e seleção, os autores devem ser contatados para serem avisados e para confirmação. Caso não haja confirmação (um bom prazo seria 2-3 dias), o próximo colocado deve ser contatado e o não-confirmado deve ser avisado disto.

Reviewer: ```suggestion
Ao finalizar a avaliação e seleção, os autores devem ser contatados para serem avisados e para confirmação. Caso não haja confirmação (um bom prazo seria 2-3 dias), o próximo colocado deve ser contatado e o não-confirmado deve ser avisado disto.

> Não cancele com possíveis colaboradores, até que você tenha a grade totalmente confirmada. E não esqueça de montar um cronograma bem definido, para ter todas essas datas de confirmação estejam claras.
```

Reviewer 2: Modificado :+1:

> Isto é feito por email, através do email oficial do PUG-PE. Providenciamos um [modelo de confirmação de atividade](languages/portuguese/arquivos/modelos/EMAIL-CONFIRMACAO-ATIVIDADE.md) :smile:

Reviewer: ```suggestion
Isto é feito por email, através do email oficial do PUG-PE. Providenciamos um [modelo de confirmação de atividade](modelos/EMAIL-CONFIRMACAO-ATIVIDADE.md) :smile:
```

Reviewer 2: Modificado!

> 

Reviewer: ```suggestion
> Aqui conterá uma descrição sobre submissão de atividades, avaliação de atividades e confirmação de atividades. Desde como fazer até onde fazer o gerenciamento dessas etapas.
```

Estamos seguindo esse padrão, de no início de cada página tem uma caixa de mensagem explicando resumidamente o que essa página conterá.

Reviewer 2: Adicionado

## Submissão de atividades

> *Caso o evento for oferecer atividades que pessoas possam submeter (por exemplo: palestras, tutoriais, etc.)*

Reviewer: Incrível a gente usando o nome *evento*. Melhor a gente fazer um review no contéudo, após esses PRs pra alinhar esses termos?

Reviewer 2: Acho que sim, vale um PR

Reviewer: \o/

### Descrição

> A descrição da etapa de submissão pode fazer muita diferença, pois é o que pode atrair ou afastar pessoas interessadas. A descrição deve ser objetiva e clara, para que as pessoas não fiquem perdidas no processo.

Reviewer: ```suggestion
A descrição da etapa de submissão pode fazer muita diferença, pois é o que pode atrair ou afastar pessoas interessadas. Por isso, faça uma descrição objetiva e clara, para que as pessoas não fiquem perdidas no processo.
```

Reviewer 2: Modificado

Algumas informações são necessárias para os interessados:

- Data e local do evento
- Breve descrição sobre como será o evento

> - O que eles estão submetendo (ex.: tutorial, palestra, etc.)

Reviewer: ```suggestion
- O que eles estão submetendo (ex.: tutorial, palestra, etc.)
    - Duração das atividades
    - Principais áreas de conhecimento que são aceitas
```

As vezes não colocamos descrição sobre a duração dessas atividades e nem quais são as grandes áreas de temas possíveis (tem no site do Pug, quando vc submete palestras, seria legal usar elas).

Reviewer 2: Sim, certo. No modelo de descrição tem isso, mas seria bom explicitar aqui, valeu

> - Instruções de como deve ser feita a submissão

Reviewer: ```suggestion
- Instruções de como deve ser feita a submissão
   - Título
   - Resumo
```

No evento anterior, não teve um padrão na descrição das palestras. Ficou ruim até pra cadastrar no site, algumas não tinha resumo direito.

Reviewer 2: Se tu olhou o Speakerfight, tinha um guideline sobre como deveria ser a descrição das palestras. Se as pessoas seguem ele ou não é outra história.

O ponto aqui não é nem sobre quais informações devem estar disponíveis para os participantes. Talvez a forma do guideline tenha que ser mais concisa. Talvez tenha que disponibilizar um template de submissão. Alguma coisa assim. Por isso prefiro deixar esse tópico aí em aberto.

Reviewer: No Python Nordeste tinha um template nera? Sim sim, seria legal isso de deixar mais conciso, se usar o forms do google é mais facil organizar isso.

Reviewer 2: Então, os guidelines foram baseados no "template" da Python Nordeste :)

Concordo, no Google Forms é bem mais simples de fazer isso

- Breve explicação do processo de avaliação e seleção
- Declaração sobre o Código de Conduta
- Divulgação dos canais de comunicação do grupo

> Aqui temos um [modelo de descrição de submissão de palestras](languages/portuguese/arquivos/modelos/DESCRICAO-SUBMISSAO-PALESTRAS.md) que pode ser utilizado. E também um [modelo de descrição de votação de palestras](languages/portuguese/arquivos/modelos/DESCRICAO-VOTACAO-PALESTRAS.md), caso seja necessário.

Reviewer: ```suggestion
Aqui temos um [modelo de descrição de submissão de palestras](modelos/DESCRICAO-SUBMISSAO-PALESTRAS.md) que pode ser utilizado. E também um [modelo de descrição de votação de palestras](modelos/DESCRICAO-VOTACAO-PALESTRAS.md), caso seja necessário.
```

Reviewer 2: Modificado :+1:
